src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -192,7 +193,8 @@ class ProfileView extends React.Component {
 
 function mapStateToProps(state) {
   return {
-    showDirectDeposit: profileShowDirectDeposit(state),
+    directDepositIsSetUp: directDepositIsSetUpSelector(state),
+    showDirectDepositFeature: profileShowDirectDeposit(state),
     showReceiveTextNotifications: profileShowReceiveTextNotifications(state),
   };
 }
```

Comment:
Not introduced as part of this PR so probably out of scope, but since this is a container, it should probably be stored in the `containers` directory. We could also probably consolidate some of its data access with the parent component, since I think that is also a container. Maybe another ticket?

Reply:
Good catch. But the only reason that it's a container right now is to access the Flipper feature flags, which live in the Redux state. when the features are turned on "for good" and the feature flags are removed, this will no longer be a connected component. I realize we don't have best practices for this yet, since Flipper is so new, but I think it makes sense to leave it in this directory for now.

Reply:
Ah gotcha! Sounds like a good plan to me then 🙂